### ISSUE TITLE
[Bug]: Home menu bar address/copy icon gap

### ISSUE THREAD

**Issue by tmashuang** [MetaMask/metamask-extension#19136]
### Describe the bug

Regression from previous version of the home menu bar there seems to be an added gap between the address and copy icon.

### Before 
![address-copy-no-gap-prod](https://github.com/MetaMask/metamask-extension/assets/13376180/5962023c-64f9-40fe-b6a3-06b9852b433d)

### After

![address-copy-gap-rc](https://github.com/MetaMask/metamask-extension/assets/13376180/26ae1b49-3568-49b6-bf15-b74c9d1d42e4)

### Steps to reproduce

1. Open home screen
2. View menu bar
3. See gap between address and copy icon vs previous/prod version
4. TLDR; See provided screenshots

### Version

v10.31.0

### Build type

None

### Browser

Chrome

### Operating system

Linux

**Comment by darkwing** [comment]
@tmashuang I cannot reproduce this locally in popup or full screen mode, nor in my personal MetaMask or `develop` branch.  Additionally, the copy icon change happened a few releases ago, so the `before` screenshot having a square copy button is interesting.  Could you use Developer Tools to inspect the space between the address and icon and let me know what element is there?  Happy to jump on a call as well to figure this out.

**Comment by tmashuang** [comment]
The gap is present in the [v10.31.0 RC](https://github.com/MetaMask/metamask-extension/pull/19032) branch, you would need to build that branch. As with the element in question, it is this when I inspect it:
```
<span class="box mm-icon mm-icon--size-md box--display-inline-block box--flex-direction-row box--color-icon-alternative" style="-webkit-mask-image: url(&quot;./images/icons/undefined.svg&quot;);"></span>
```

**Comment by tmashuang** [comment]
Found it, solved by #19087. RC is going to need that commit.
